src/Sepes.Infrastructure/Service/Sandbox/SandboxService.cs

```diff
@@ -150,6 +151,14 @@ public async Task<SandboxDetailsDto> CreateAsync(int studyId, SandboxCreateDto s
             }
```

Comment:
Codeql does not like generic catch clauses, do you have any suggestions of an alternative?